addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/backend/ReadResource.java

```diff
@@ -156,10 +156,10 @@ public Object getStates(@QueryParam("a") List<String> itemNames, @QueryParam("i"
                     }
                     states.add(itemState);
                 } catch (ItemNotFoundException e) {
-                    logger.error(e.getLocalizedMessage());
+                    logger.error("{}", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace